app/models/meeting_option.rb

```diff
@@ -12,4 +12,11 @@ def self.bbb_options(room_id:)
       .where(room_meeting_options: { room_id: })
       .pluck(:name, :value)
   end
+
+  def self.find_room_meeting_option_by_name(name:, room_id:)
+    joins(:room_meeting_options)
+      .where(room_meeting_options: { room_id: })
+      .select('meeting_options.name', 'room_meeting_options.value')
```

Comment:
This is due to `ActiveRecord` having a hard time to deal with joins so it assumes that the result since it's initiated by the `MeetingOption` model it will result in an instance of `MeetingOption` so it will filter any irrelevant attributes such us `value`.